src/components/containers/UpOnlyWBTC.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from "react";
+import ExperimentDetail from "../ExperimentDetail";
+import stakedPut from "../../contracts/stakedPut";
+
+class UpOnlyWBTC extends Component<
```

Comment:
Do you mind if we stick to using React functional components? It helps with readability